drivers/spi/sunxi_spi.c

```diff
@@ -254,10 +254,11 @@ static int sunxi_spi_xfer(struct udevice *dev, unsigned int bitlen,
 		sunxi_spi_cs_activate(dev, slave_plat->cs);
 
 	while (len) {
-		nbytes = min(len, (size_t)64 - 1);
+		nbytes = min(len, (size_t) 64);
```

Comment:
As far as I am aware this doesn't work for SPI2 on the A10 (see drivers/spi/spi-sun4i.c in the Linux kernel source).